scripts/build_aarch64_container.sh

```diff
@@ -17,8 +17,8 @@ GOLANG_VERSION=@sha256:f7d3519759ba6988a2b73b5874b17c5958ac7d0aa48a8b1d84d66ef25
 ALPINE_VERSION=@sha256:b06a5cf61b2956088722c4f1b9a6f71dfe95f0b1fe285d44195452b8a1627de7
 # mcr.microsoft.com/dotnet/sdk:6.0-focal
 DOTNET_BUILDER=@sha256:749439ff7a431ab4bc38d43cea453dff9ae1ed89a707c318b5082f9b2b25fa22
-# node-package-builder-glibc - centos7:latest
-NODE_PACKAGE_BUILDER_GLIBC=@sha256:65a4aad1156d8a0679537cb78519a17eb7142e05a968b26a5361153006224fdc
+# node-package-builder-glibc - centos8
```

Comment:
`centos8:latest`?

If we build on CentOS 8, does it load on CentOS 7?

Reply:
It loads because we are building with: `-static-libstdc++`

Reply:
I am not sure if this is centos8:latest though. I took image that is already used in build_aarch64_executable as base for gprofiler_builder. I'll change it to centos8:latest and check if it will still work.

Reply:
Changed to centos8:latest and it works.